.github/pull_request_template.md

```diff
@@ -10,11 +13,11 @@ This PR fixes #
 
 - [ ] Yes, I am a Member of **DEVSTRONS'**
 
-**Discord id** (eg: DEVSTRONS'#0238):
-<!-- Enter your Discord's ID -->
+**Discord ID** (e.g.: DEVSTRONS'#0238)
 
+- Your Discord ID <!-- Enter your Discord ID -->
```

Comment:
@Panquesito7: I already gave the example, no need for this extension of keywords.
```suggestion
<!-- Enter your Discord ID -->
```